### PR TITLE
Fix sticky navbar regression

### PR DESCRIPTION
**Pull request by EmmanuelVella**
```markdown
### Fixed
- Fixed sticky navbar regression
```

## Subject

As @dmarkowicz [noticed in my previous PR](https://github.com/sonata-project/SonataAdminBundle/pull/4675#issuecomment-333204372), the `debounce` function was added to improve performances. Removing it was a mistake !

My PR was merged before I fixed it, so here is another one with the `debounce` function reintroduced.  I have adjusted the code to make it correctly work with it.

### CODE REVIEW

File: Resources/public/Admin.js [hunk 1]
```diff
@@ -570,39 +570,39 @@ var Admin = {
...
+                    jQuery(navbar).removeClass('stuck');
+                }
+            }, 250)
+        );
 
         jQuery('body').on('expanded.pushMenu collapsed.pushMenu', function() {
-            Admin.handleResize(footer, navbar, wrapper, topNavbar);
+            setTimeout(function() {
+                Admin.handleResize(footer, navbar, wrapper);
+            }, 350); // the animation take 0.3s to execute, so we have to take the width, just after the animation ended
```
Comment (greg0ire) [inline Resources/public/Admin.js]: "takes"

Comment (EmmanuelVella) [reply Resources/public/Admin.js]: Fixed

**Comment by greg0ire** [comment]
Thanks @EmmanuelVella

**Comment by EmmanuelVella** [comment]
👍

**Pull request merged**